### PR TITLE
Add missing Sources and Destinations to Catalog Table.

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Changes
We are missing a few sources and destinations from the catalog table, so I updated it to include them.
Some examples:
- S3
- Amplitude
- Google Ads

## Misc Changes
- I upgraded Facebook Marketing to Certified, as I think we did recently upgrade and certify it, @sherifnada can you confirm?

### CODE REVIEW

File: docs/integrations/README.md [hunk 1]
```diff
@@ -13,17 +13,21 @@ Airbyte uses a grading system for connectors to help users understand what to ex
...
 |[Appstore](./sources/appstore.md)| Alpha |
 |[Asana](./sources/asana.md) | Beta |
+|[AWS CloudTrail](./sources/aws-cloudtrail.md)| Beta |
 |[Braintree](./sources/braintree.md)| Alpha |
 |[ClickHouse](./sources/clickhouse.md)| Beta |
+|[Db2](./sources/db2.md)| Beta |
 |[Drift](./sources/drift.md)| Beta |
 |[Exchange Rates API](./sources/exchangeratesapi.md)| Certified |
-|[Facebook Marketing](./sources/facebook-marketing.md)| Beta |
+|[Facebook Marketing](./sources/facebook-marketing.md)| Certified |
```
Comment (sherifnada) [inline docs/integrations/README.md:25]: I would keep as beta for a few weeks -- we're still working out kinks espeically with insights streams

File: docs/integrations/README.md [hunk 2]
```diff
@@ -52,6 +55,7 @@ Airbyte uses a grading system for connectors to help users understand what to ex
 |[Plaid](./sources/plaid.md)| Alpha |
 |[PokéAPI](./sources/pokeapi.md)| Beta |
 |[Postgres](./sources/postgres.md)| Certified |
+|[PostHog](./sources/posthog.md)| Certified |
```
Comment (sherifnada) [inline docs/integrations/README.md:58]: recently released so should be beta

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/README.md (hunk 1 above)
Comment (avaidyanatha) [reply docs/integrations/README.md]: Done!

File: docs/integrations/README.md (hunk 2 above)
Comment (avaidyanatha) [reply docs/integrations/README.md]: Done!

**Pull request merged**